### PR TITLE
Update ua.json

### PR DESCRIPTION
**Pull request by Ro-off**

### CODE REVIEW

File: langs/ua.json [hunk 1]
```diff
@@ -31,12 +31,22 @@
...
     "CONNECTION_FAILED": "Не вдалося підключитися",
-    "LOST_CONNECTION": "З’єднання втрачено",
+    "LOST_CONNECTION": "З'єднання з сервером втрачено",
     "UNKNOWN_ERROR": "Невідома помилка",
     "WRONG_PASSWORD": "Невірний пароль",
-    "DEBUG_NOT_ALLOWED": "Режим налагодження недоступний",
+    "DEBUG_NOT_ALLOWED": "Режим налагодження недоступний на цьому сервері",
     "WRONG_BRANCH": "Сервер використовує іншу вітку оновленнь",
     "WRONG_STABLE_BUILD": "Сервер використовує іншу версію alt:V",
     "WRONG_BETA_BUILD": "Сервер використовує новішу версію alt:V. Перезавантажте клієнт для оновлення"
```
Comment (ccrumbb) [inline langs/ua.json]: coma at EOL missing

File: langs/ua.json [hunk 2]
```diff
@@ -31,12 +31,22 @@
...
+    "CLIENT_OUTDATED": "Сервер використовує новішу версію alt:V",
+    "SERVER_OUTDATED": "Сервер використовує застарілу версію alt:V"
```
Comment (ccrumbb) [inline langs/ua.json]: coma at EOL missing

**Comment by vadzz-dev** [comment]
Pls properly format json file before merge

**Comment by vadzz-dev** [comment]
if you're not familar with JSON i suggest you to use https://jsonlint.com/

**Comment by Ro-off** [comment]
Thanks, i hope now it valid

**Pull request merged**